cmd/kubeadm/app/master/manifests.go

```diff
@@ -367,6 +362,24 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted bool) [
 	return command
 }
 
+func getEtcdCommand(cfg *kubeadmapi.MasterConfiguration) []string {
+	listenURL := "http://127.0.0.1:2379"
+	if cfg.Etcd.ListenURL != "" {
+		listenURL += fmt.Sprintf(",%s", cfg.Etcd.ListenURL)
+	}
+	advertiseURL := "http://127.0.0.1:2379"
+	if cfg.Etcd.AdvertiseURL != "" {
+		advertiseURL += fmt.Sprintf(",%s", cfg.Etcd.AdvertiseURL)
+	}
+
```

Comment:
I wasn't 100% sure what you meant, but I added a new default variable which both URLs point to. That DRYed it up a bit. Is that what you meant?